### ISSUE TITLE
JsonElement.toString() is not consistent with stringify on default settings

### ISSUE THREAD

**Issue by qwwdfsad** [Kotlin/kotlinx.serialization#323]
```
@Serializable
data class Primitive(val primitive: Int = 42)

println( json { "primitive" to 42}
println(Json.stringify(Primitive()))
```

will print
```
{"primitive": 42}
{"primitive":42}
```

**Comment by q-litzler** [comment]
I'm very curious to know which one you consider to be desirable ? With or without a space ? 😄

**Comment by sandwwraith** [comment]
Fixed by #325
